.travis.yml

```diff
@@ -0,0 +1,16 @@
+language: python
+sudo: false
+python:
+  - "2.7"
+env:
+  - TOX_ENV=docs
+install:
+  - pip install -rrequirements/docs.txt
+script:
+  - cd docs && make html SPHINXOPTS='-nW'
```

Comment:
there's the `-n` here already (http://sphinx-doc.org/invocation.html#cmdoption-sphinx-build-n), were you thinking about something else?